router/transformer/transformer.go

```diff
@@ -64,7 +64,7 @@ func loadConfig() {
 	config.RegisterIntConfigVariable(30, &maxRetry, true, 1, "Processor.maxRetry")
 	config.RegisterDurationConfigVariable(100, &retrySleep, true, time.Millisecond, []string{"Processor.retrySleep", "Processor.retrySleepInMS"}...)
 	config.RegisterDurationConfigVariable(30, &timeoutDuration, false, time.Second, []string{"HttpClient.timeout"}...)
-	config.RegisterInt64ConfigVariable(15, &retryWithBackoffCount, true, 1, "Router.transformerProxyRetryCount")
+	config.RegisterInt64ConfigVariable(3, &retryWithBackoffCount, true, 1, "Router.transformerProxyRetryCount")
```

Comment:
Let's remove the retry logic completely and rely on the router's retry mechanism. It'll also align with the previous behavior.

Reply:
Yes, working on it 🧨